docs/ModStorage/index.md

```diff
@@ -0,0 +1,61 @@
+# Mod Storage
+
+Read [Mod Structure](../ModStructure/index.md) if you haven't done so already to set up the storage version.
+
+```csharp
+
+```
+
+## 
+
+```csharp
+
+modded class KitBase // extends from ItemBase
+{
+	int someIntVariable;
+	string someStringVariable;
+
+	override void CF_OnStoreSave(map<string, CF_ModStorage> storage)
+	{
+		//! Always call super at the start.
+		super.CF_OnStoreSave(storage);
+
+		auto ctx = storage["JM_CommunityFramework"];
```

Comment:
Why does it call `storage["JM_CommunityFramework"];`? Is this supposed to be a named storage and this is an "example mod name"?

Reply:
Will add further documentation to make clear